tailcfg/tailcfg.go

```diff
@@ -1964,10 +1964,11 @@ const (
 	// Funnel warning capabilities used for reporting errors to the user.
 
 	// CapabilityWarnFunnelNoInvite indicates whether Funnel is enabled for the tailnet.
-	// NOTE: In transition from Alpha to Beta, this capability is being reused as the enablement.
+	// This cap is no longer used in client versions released after 2023-08-03.
```

Comment:
date is also fine, but we prefer to mention the capVer. Do you need to bump the capVer for this, if this is the commit that stopped using this? Or did the earlier commit do it?

Reply:
So for this one, the control server actually stopped sending it a while back, but not totally sure when. But we were still checking it in the client code until now. Anyways, yeah let me bump the capVer.